ZenPacks/zenoss/WBEM/bin/mof_compiler

```diff
@@ -0,0 +1,4 @@
+#!/opt/zenoss/bin/python2.7
```

Comment:
None of this stuff that `make build` installs into ZenPacks/zenoss/WBEM/(bin|lib) should be added to git.

Reply:
It was never intentionally added, (not sure how it got in.)  Probably accidentally when I was bulk-adding the libs under ZenPacks/zenoss/WBEM/lib.  They're gone